convert_lora_to_gguf.py

```diff
@@ -363,7 +363,11 @@ def get_tensors(self) -> Iterator[tuple[str, Tensor]]:
                     yield (name, cast(torch.Tensor, LoraTorchTensor(tensor.A, tensor.B)))
 
             def modify_tensors(self, data_torch: Tensor, name: str, bid: int | None) -> Iterable[tuple[str, Tensor]]:
-                dest = super().modify_tensors(data_torch, name, bid)
+                dest = list(super().modify_tensors(data_torch, name, bid))
```

Comment:
Is it needed to recast to a list `super().modify_tensors(data_torch, name, bid)`?

Reply:
Without this, pyright will not pass: https://github.com/ggerganov/llama.cpp/actions/runs/10472198558/job/29001100132

Reply:
Yes, because it can also be a `Generator` (when `modify_tensors` uses `yield`)